src/options.js

```diff
@@ -67,6 +68,11 @@ export default {
   optOut: false,
   onError: () => {},
   onExitPage: () => {},
+  plan: {
```

Comment:
Could we default this to `undefined` instead?

Reply:
yeah, was thinking about this, hesitate about if we should have the whole plan to be null or undefined by default. Couple reasons leading to this, one is to give a basic example for what the plan should be like, another is for we have a method in AmplitudeClient called `_parseConfig`, which will help check and assert types for certain configs, so it will also help to check the input and sanity, but it will need to define plan as object and also listed the three properties with a default string.

Reply:
@qingzhuozhen got it, that makes sense.

Reply:
Ok got it. That makes sense.